src/langsmith/faq.mdx

```diff
@@ -41,9 +41,46 @@ Our deployment only needs egress for a few things (most of which can reside with
 
 Your VPC can set up rules to limit any other access. Note: We require the `X-Organization-Id` and `X-Tenant-Id` headers to be allowed to be passed through to the backend service. These are used to determine which organization and workspace (previously called "tenant") the request is for.
 
-### *Resource requirements for the application?*
+## *Resource requirements for the application?*
 
 * In kubernetes, we recommend a minimum helm configuration which can be found in [here](https://github.com/langchain-ai/helm/blob/main/charts/langsmith/examples/medium_size.yaml). For docker, we recommend a minimum of 16GB of RAM and 4 CPUs.
 * For Postgres, we recommend a minimum of 8GB of RAM and 2 CPUs.
 * For Redis, we recommend 4GB of RAM and 2 CPUs.
 * For Clickhouse, we recommend 32GB of RAM and 8 CPUs.
+
+## SAML SSO FAQs
+
+### *How do I change a SAML SSO user's email address?*
+
+Some identity providers retain the original `User ID` through an email change while others do not, so we recommend that you follow these steps to avoid duplicate users in LangSmith:
+
+1. Remove the user from the organization (see [here](/langsmith/set-up-a-workspace#manage-users))
+2. Change their email address in the IdP
+3. Have them login to LangSmith again via SAML SSO - this will trigger the usual [JIT provisioning](#just-in-time-jit-provisioning) flow with their new email address
```

Comment:
```suggestion
3. Have them log in to LangSmith again via SAML SSO - this will trigger the usual [JIT provisioning](#just-in-time-jit-provisioning) flow with their new email address
```